build.gradle

```diff
@@ -1,13 +1,16 @@
 /*
- * This build file was generated by the Gradle 'init' task.
- *
- * This generated file contains a sample Java project to get you started.
- * For more details take a look at the Java Quickstart chapter in the Gradle
- * user guide available at https://docs.gradle.org/3.5/userguide/tutorial_java_projects.html
+ * This file defines the steps to build Synthea using Gradle. Full documentation of the syntax
+ * can be found at https://docs.gradle.org/6.5/userguide/java_plugin.html
  */
 
 // Apply the java plugin to add support for Java
 apply plugin: 'java'
+
```

Comment:
Can I request the two compatibility lines also be added outside the `compileJava` scope as well? Not sure if it's an eclipse bug, but if those aren't included, it picks the wrong target JDK when you import, and the build fails because some XML stuff isn't included as of Java 9. I don't think it should have any other impact.

i.e,
```diff
+sourceCompatibility = '1.8'
+targetCompatibility = '1.8'
+
+compileJava {
+    sourceCompatibility = '1.8'
+    targetCompatibility = '1.8'
+}
```